ops/configuration/variables/env-sample

```diff
@@ -46,7 +46,7 @@ YII2_PATH=/var/www/vendor/yiisoft/yii2
 DATA_SAVE_PATH=~/.containers-data/default-gigadb
 YII_DEBUG=true
 YII_TRACE=3
-DISABLE_CACHE=true
+DISABLE_CACHE=false
```

Comment:
**question:** Would it be better to keep DISABLE_CACHE equal to `false`? This might be better for `dev` environment so that us developers can make changes in the database and they are  immediately displayable in GigaDB web pages. For testing this PR, we just manually change this variable in .env.

Reply:
After discussion, since the invalidation query is working, and all the test cases about the caching is passing, caching will be on by default for `dev` environment as well.